x/staking/client/rest/tx.go

```diff
@@ -31,35 +31,35 @@ func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec
 }
 
 type (
-	// MsgBeginRedelegateInput defines the properties of a delegation request's body.
-	MsgDelegationsInput struct {
+	// DelegateRequest defines the properties of a delegation request's body.
+	DelegateRequest struct {
 		BaseReq          rest.BaseReq   `json:"base_req"`
 		DelegatorAddress sdk.AccAddress `json:"delegator_address"` // in bech32
 		ValidatorAddress sdk.ValAddress `json:"validator_address"` // in bech32
 		Delegation       sdk.Coin       `json:"delegation"`
```

Comment:
what about switching this to `amount` for consistency with the other msgs ?

Reply:
++

Reply:
ACKed.